docs/source/whats-new.rst

```diff
@@ -13,7 +13,9 @@ Bug fixes
 
 - Fixed plugin batchmode submit bug. Batchmode plugins did not pick up the
   correct evaluation system config file. This has been fixed now.
-
+- Corrected minor typos in documentation and code comments. (fix typo, minor typo, minor typo separator)
```

Comment:
Would be better to change it to: Revised minor typographical errors in the documentation and code comments.

Reply:
Ok